minikube.yml

```diff
@@ -0,0 +1,137 @@
+# This file is for testing
+# I'm trying to deploy lifebank in MiniKube
+# Any help is welcome
```

Comment:
@Edgar097 whats you goal here?

Reply:
@kuronosec I want a functioning lifebank app in minikube, where all the pods interact with each other as they do for example in the actual version, however is not working properly.